oscm-portal/WebContent/marketplace/templates/main.xhtml

```diff
@@ -188,6 +192,7 @@
     AdmUtils.contentsTitle();
     AdmUtils.setInfoPanel("progressMessages");
     AdmUtils.setInfoPanel("infoMessages");
+<<<<<<< HEAD
```

Comment:
Thanks for modification of pull request. I see you have rebased/merged master to branch. Are you sure that lines 195, 210, 212 should be contained in the file?